packages/dds/SchemaVersioning.md

```diff
@@ -38,7 +38,7 @@ This document makes the following assumptions which relate to use of these schem
         This means two different "current" versions must be able to have different Fluid Framework versions.
 
 1.  Collaboration between users should work as long as they have a version of the application that is "current" for a definition of "current" controlled by the application authors (not the Fluid Framework authors!).
-    This includes cross version collaboration between multiple current versions of the application with different Fluid Framework versions noted above.
+    This includes cross-client collaboration between multiple current versions of the application with different Fluid Framework versions noted above.
```

Comment:
In places that refer to "cross version collaboration" before, I think you should change the whole phrase ("cross-client compatibility") rather than just the version->client part. This sentence is probably reasonable enough that one can infer its meaning, but "cross-client collaboration" just sounds redundant and there are some other usages where it's less obvious

Reply:
Makes sense, updated in latest